### PR TITLE
Create works and about myself sections

### PR DESCRIPTION
**Pull request by naji-97**
I finished my works section and about myself section

### CODE REVIEW

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @naji-97. 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not pass one of the linters checks. Kindly fix the linter error and submit it again.🙏

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by huseyinbiyik** [comment]
Hi @naji-97 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

-You have submitted a project, that has no new Readme templated used. Kindly replace your Readme file with the new one and be sure that you deleted all the placeholder text and replace with project info. Please check this [link](https://github.com/microverseinc/readme-template) for the new template.
-Kindly add Pull Request description including what changes made on this branch. Please check this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=Write%20a%20meaningful%20title%20for%20your%20pull%20request%20and%20always%20add%20a%20short%20summary%20of%20changes%20introduced%20in%20the%20pull%20request.) to learn how to do it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -13,28 +13,149 @@
...
+   <li><a title="github" href="https://github.com/naji-97"><img src="./assest/icons/Vector.svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (3).svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (4).svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (2).svg" alt="" srcset=""></a></li>
+  </ul>
+ </section>
+
+  
+
+ <section class="projects">
```
Comment (DuaneDave) [inline index.html:41]: Kindly use grid to display the contents in this section as per the requirements to make your design process easy.

File: index.html [hunk 2]
```diff
@@ -13,28 +13,149 @@
...
+  <p class="decription">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+   <ul class="language">
+    <li>html</li>
+    <li>css</li>
+    <li>js</li>
+   </ul>
+   <div class="see-project">
+    <p><a href="#">See project</a></p>
+   </div>
+  </div>
```
Comment (DuaneDave) [inline index.html]: Please align the contents to match the Figma template with the right spacing and color for the header and paragraph to make your page more visually appealing.

The color code for the header is `#172B4D`
for paragraph `#344563`

consider using `flex` as per the requirements (do the same for all other cards).


Your design 👇👇👇


not aligned


Figma design 👇👇👇



Figmaaaaa

File: index.html [hunk 3]
```diff
@@ -13,28 +13,149 @@
...
+  </ul>
+  <p class="decription">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+   <ul class="language">
+    <li>html</li>
+    <li>css</li>
+    <li>js</li>
+   </ul>
+   <div class="see-project">
+    <p><a href="#">See project</a></p>
+   </div>
```
Comment (DuaneDave) [inline index.html]: Kindly use a button tag for the see project and style the various states (hover, active and disabled (where necessary)) as required in the project to make it interactive.

see here...


btn states

File: index.html [hunk 4]
```diff
@@ -13,28 +13,149 @@
...
+  <ul class="links">
+   <li><a title="linkdin" href="https://www.linkedin.com/in/naji-ali-55b5421b3/"><i class="fa-brands fa-linkedin fa-2x"></i></a></li>
+   <li><a title="github" href="https://github.com/naji-97"><img src="./assest/icons/Vector.svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (3).svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (4).svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (2).svg" alt="" srcset=""></a></li>
+  </ul>
+  <div class="see-resume">
+   <p><a href="#">See resume</a></p>
+  </div>
```
Comment (DuaneDave) [inline index.html]: As I commented before, kindly add the various states for this element As optional, I suggest you use a button tag to make styling it faster.

Comment (DuaneDave) [inline index.html]: Please use the correct color for the texts and header tags and also consider spacing the contents correctly to match the template.

The color code for the header is `#172B4D`
for paragraph `#344563`

As a side note, please add the correct images for the languages as well.


Your design 👇 


style proper




Figma design 👇



Proper design

File: style.css [hunk 5]
```diff
@@ -12,6 +12,10 @@ body {
   color: #000;
   font-family: "Poppins", sans-serif;
   width: 400px;
+
+  /* padding-left: 120px;
+  padding-right: 120px; */
+
   margin: auto;
```
Comment (DuaneDave) [inline style.css]: Please do not leave behind commented pieces of code as this clutters the clarity of your code base.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @naji-97  👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Clearly structured code
✅ Good commit history
✅ Nice usage of semantic tags


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -13,28 +13,149 @@
...
+       <li>javaScript</li>
+      </ul>
+      <div class="see-project">
+       <button><a href="#">See project</a></button>
+      </div>
+     </div>
+ </section>
+ <section class="about">
+  <div class="content">
+  <h2>About Myself</h2>
```
Comment (ridwanediallo) [inline index.html]: - [ ] Please as requested by the previous reviewer make sure that the `About  Myself` heading is in two lines like in the Figma design. 

### your design 
Screenshot from 2022-12-03 20-53-48
### The figma dedign
Screenshot from 2022-12-03 20-54-32

### One way to do that is to add <br /> between the About and Myself. Like this 👇️

` <h2>About <br /> Myself</h2>`

File: index.html [hunk 7]
```diff
@@ -13,28 +13,149 @@
...
+  <p class="description">
+   Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+  </p>
+  <h5>LET'S CONNECT</h5>
```
Comment (ridwanediallo) [inline index.html:126]: - [OPTIONAL ] Please make sure that the colour of your `LET'S CONNECT` is the same compared to the one on Figma.

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @naji-97 , 


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @naji-97 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**